sympy/printing/pretty/tests/test_pretty.py

```diff
@@ -332,6 +332,11 @@ def test_upretty_modifiers():
     assert upretty( Symbol('x__dot') ) == u('x__dot')
 
 
+def test_pretty_Cycle():
+    from sympy.combinatorics.permutations import Cycle
+    assert pretty(Cycle(1, 2)) == '([1, 2],)'
```

Comment:
I think i am confused now. I quote this from [here](http://docs.sympy.org/dev/tutorial/printing.html#str)
> String forms are designed to be easy to read, but in a form that is correct Python syntax so that it can be copied and pasted. The str() form of an expression will usually look exactly the same as the expression as you would enter it.

So my understanding is `str` printing is the same as `print` printing. Now in the above quote, it is said that: ` in a form that is correct Python syntax so that it can be copied and pasted.`. But you said to: ""So it seems that the version of _print_Cycle currently in str.py should be moved to repr.py and replaced with a new printer.""
Currently in master-sympy
```
>>> print(Cycle(1, 2))
Cycle(1, 2)
```
So introduction of a new-printer is expected to print it as `(1 2)`?